src/modules/tokens/models/tokens.filter.args.ts

```diff
@@ -9,3 +20,22 @@ export class TokensFiltersArgs {
     @Field({ defaultValue: false })
     enabledSwaps: boolean;
 }
+
+@InputType()
+export class TokensFilter {
+    @Field(() => [String], { nullable: true })
+    identifiers: string[];
+    @Field({ nullable: true })
```

Comment:
You can add also Field type in @Field decorator

Reply:
Done